supplementary_style_guide/style_guidelines/specific-doc-types.adoc

```diff
@@ -5,5 +5,117 @@
 [[release-notes]]
 == Release notes
 
+[[release-notes-doc-texts]]
+=== Conventions for phrasing Release Notes doc texts
+
+A doc text is a short description of an engineering Bugzilla or Jira issue that is published in product Errata Advisories and Release Notes. Engineering supplies draft content, which is a summary of the issue, then technical writers edit or rewrite the draft content in accordance with _The IBM Style Guide_. You can write doc texts in more than one way, depending on the issue type: 
+
+* Enhancements
+* Bug Fixes
```

Comment:
Since we use sentence case in CCS, I'd use that for this list of types, and in the corresponding headings below. Except "Technology Preview", which we standardized to always capitalize.

Reply:
These doc types are capitalized in Bugzilla, so I kept them in caps here. Would agree with keeping them this way in that case?
But by that logic they are gui elements really, in which case they should also be in bold font.  
WDYT @bergerhoffer ?

Reply:
@ndeevy Hmm I see what you're getting at. Though if I look at it in Bugzilla, then it should be "Enhancement" and "Bug Fix" singular. However, since this says that it could be for either Bugzilla or Jira (or any other tracking system really), I don't think that we need to match what Bugzilla uses, so I'd still go with sentence case in my opinion.